modules/commons/src/test/java/org/apache/synapse/commons/staxon/core/event/SimpleXMLEventFactoryTest.java

```diff
@@ -235,10 +235,7 @@ public void testCreateProcessingInstruction() throws XMLStreamException {
 
     @Test(expected = UnsupportedOperationException.class)
     public void testCreateDTD() {
-        try {
-            factory.createDTD("foo");
-        } catch (UnsupportedOperationException ex) {
-            //empty
-        }
+        factory.createDTD("foo");
+        Assert.fail("executed without expected exception");
```

Comment:
we do not need a assert.fail since this statement not executed because of factory.createDTD("foo") throws the exception. It it failed to throw exception, junit will fail the test